src/presets.js

```diff
@@ -237,6 +237,58 @@ module.exports = {
 			}
 		}
 
+		presets['terminate_apps'] = {
+			category: 'App',
+			name: `Terminate Apps`,
```

Comment:
Assuming it will close any open apps:
```suggestion
			name: `Terminate All Apps`,
```